ocs_ci/helpers/dr_helpers_ui.py

```
@@ -915,13 +959,163 @@ def check_or_assign_drpolicy_for_discovered_vms_via_ui(
+        switcher_found = False
+        try:
+            switcher_el = acm_obj.driver.find_element(
+                By.XPATH,
+                "//button[@data-test-id='perspective-switcher-toggle']"
+                " | //*[contains(@class, 'perspective-switcher')]//button",
+            )
+            switcher_found = True
+            if "fleet virtualization" in switcher_el.text.strip().lower():
+                still_in_fleet_virt = True
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)